puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/source/hammer-cli-x-test.groovy

```diff
@@ -34,10 +34,20 @@ pipeline {
                 run_test(ruby: '2.5')
             }
         }
-        stage("Release Nightly Package") {
+        stage('Build and Archive Source') {
```

Comment:
shall this file still be called `…test.groovy` given it's more a source release?

Reply:
Did you mean name it to source rather than test?